slyd/.gitignore

```diff
@@ -20,3 +20,6 @@ data/*
 # npm files
 node_modules/*
 npm-debug.log
+
+#ember build
+dist/*
```

Comment:
We are shipping the compiled assets to users in the dist folder.

The command below is used to build the updated js:

    `ember build -e production`